services/storage/src/simcore_service_storage/modules/celery/_task.py

```diff
@@ -77,41 +52,88 @@ def wrapper(task: AbortableTask, *args: P.args, **kwargs: P.kwargs) -> R:
     return decorator
 
 
+def _error_handling(max_retries: NonNegativeInt, delay_between_retries: timedelta):
+    def decorator(func: Callable[..., Any]) -> Callable[..., Any]:
+        @wraps(func)
+        def wrapper(task: Task, *args: Any, **kwargs: Any) -> Any:
+            try:
```

Comment:
please adjust the typing and use TypeVar then. this way the linters are using the correct types and not some Any that do not resolve to anything.

something along these lines
```python
T = TypeVar("T")
P = ParamSpec("P")
R = TypeVar("R")

def _error_handling(...) -> Callable[[Callable[P,R]],Callable[P,R]]:
    def _decorator(func: Callable[P,R]) -> Callable[P,R]:
        def _wrapper(task: Task, *args: P.args, **kwargs: P.kwargs) -> R:
```